src/sequencer/sequencer.cpp

```diff
@@ -181,7 +182,7 @@ Sequencer::Bar Sequencer::click(Run run)
     const auto mspb = 1000.f / bps; // ms per beat
     const auto mspc = mspb / (float)midier::Time::Subdivisions; // ms per click
 
-    if (_clicked == -1)
+    if (_clicked == ULLONG_MAX)
```

Comment:
we initialize `_clicked` to `-1` explicitly, so why not checking against this value as well?

Reply:
Because `_clicked` is a unsigned long long and therefore can't be compared to -1 without throwing a warning. This is to remove the warning.